docker-compose.yml

```diff
@@ -14,7 +13,10 @@ services:
     volumes:
       - ./ezxssdb:/var/lib/mysql
   ezxss:
-    build: ./docker-php
+    build:
+      context: .
+      dockerfile: ./docker-php/Dockerfile
+      target: partial
```

Comment:
Note, the docker-compose references the partial container to keep backwards compatibility.